docs/client/client.md

```diff
@@ -0,0 +1,131 @@
+# ClientManager
+
+## ``class discodo.ClientManager(**kwargs)``
+
+This class manage voice client
```

Comment:
```suggestion
This class represents voice client
```

Reply:
`manage` is correct.

`ClientManager` don't represent voice client.